lib/stake.js

```diff
@@ -0,0 +1,78 @@
+import { toast } from "react-toastify"
+import handleTxError from "./handleTxError"
+
+export const stakeAll = async (
+  contract,
+  nftContract,
+  tokensToStake,
+  address,
+  onSuccess,
+  setPendingTxStep,
+) => {
+  try {
+    setPendingTxStep(1) // APPROVING
+
+    setPendingTxStep(2) // STAKING
+    const tx = await contract.stake(tokensToStake)
+    await tx.wait()
+    toast.success("Staked all tokens!")
+    return onSuccess()
+  } catch (error) {
+    handleTxError(error)
+    return { error }
+  }
+}
+
+export const stake = async (contract, tokenId, nftContract, onSuccess, setPendingTxStep) => {
+  try {
+    console.log("contract", contract)
+    setPendingTxStep(1) // APPROVING
+    // const approved = await isPillStakeApproved(nftContract, address, contract)
+    // if (!approved) {
+    //   const nowApproved = await approve(nftContract, contract, tokenId)
+    //   if (!nowApproved || nowApproved.error) {
+    //     return nowApproved
+    //   }
+    // }
```

Comment:
if this is not being used should just remove it.

Reply:
removed all unused functions from `lib/stake.js` 👍 

removed:
- `stakeAll`
- `claim`
<img width="925" alt="Screenshot 2023-03-01 at 1 45 56 PM" src="https://user-images.githubusercontent.com/23249402/222205994-2a50b99f-51da-4733-8110-b08c9d1a54f6.png">